Tests/SwiftDocCTests/Infrastructure/DocumentationBundleInfoTests.swift

```diff
@@ -414,4 +414,60 @@ class DocumentationBundleInfoTests: XCTestCase {
             )
         )
     }
+
+    func testFeatureFlags() throws {
+        let infoPlistWithFeatureFlags = """
+        <plist version="1.0">
+        <dict>
+            <key>CFBundleDisplayName</key>
+            <string>Info Plist Display Name</string>
+            <key>CFBundleIdentifier</key>
+            <string>com.info.Plist</string>
+            <key>CFBundleVersion</key>
+            <string>1.0.0</string>
+            <key>CDExperimentalFeatureFlags</key>
+            <dict>
+                <key>ExperimentalOverloadedSymbolPresentation</key>
```

Comment:
Should we validate that all the `CDExperimentalFeatureFlags` elements are known flags? I could imagine a developer making a slight type (e.g. a trailing whitespace) that makes stops the feature flag from parsing.

Reply:
In other words, double-check that all the given keys are ones that we recognize? Do we do that for any of the other fields in the Info.plist, e.g. the platform names? I'm worried about a warning like this breaking forward compatibility if someone uses a newer feature flag with an older version of Swift-DocC.

Reply:
It would only be a warning which doesn't break the build. 

If someone uses a new flag with an older version of DocC it would be nice to inform them that the flag isn't recognized and therefore doesn't enable anything.